qutebrowser/config/qtargs.py

```diff
@@ -259,3 +259,10 @@ def init_envvars() -> None:
                    if qtutils.version_check('5.14', compiled=False)
                    else 'QT_AUTO_SCREEN_SCALE_FACTOR')
         os.environ[env_var] = '1'
+
+    for var in config.val.qt.environ:
+        val = config.val.qt.environ[var]
+        if val == 'None':
```

Comment:
This checks against the string "None" instead of checking against the [`None` constant](https://docs.python.org/3/library/constants.html#None) from Python. Instead, this should be something like:

```suggestion
        if val is None:
```

Reply:
I did this because when I start qutebrowser using the command

`qutebrowser --temp-basedir -s qt.environ '{"QT_SCALE_FACTOR": None}`

for example I noticed that the literal None is never matched in this if statement however the string 'None' is so I assumed there was some sort of preprocessing that converts the None to 'None' before it got to this point in the code. Is this a problem or am I just misunderstanding how this is supposed to work?

Reply:
I think you'll get the expected behavior via `qutebrowser --temp-basedir -s qt.environ '{"QT_SCALE_FACTOR": null}'` (`null` rather than `None`), because YAML is used to parse config options from text. This certainly should be documented better, though.